contracts/RewardsDistributorDelegate.sol

```diff
@@ -435,6 +469,38 @@ contract RewardsDistributorDelegate is RewardsDistributorDelegateStorageV1, Expo
         setCompBorrowSpeedInternal(cToken, compSpeed);
     }
 
+    /**
+     * @notice Set borrower PCT
+     */
+    function _setBorrowerPCT(uint256 _borrowerPCT) public {
+        require(msg.sender == admin, "only admin can set comp speed");
+        borrowerPCT = _borrowerPCT;
+        supplierPCT = TOTAL_PCT.sub(_borrowerPCT);
+    }
+
+    /**
+     * @notice Set new borrow / supply speed
+     * @param cToken The market whose COMP speed to update
+     */
+    function _updateSpeedWithNewEpoch(CToken cToken) public {
```

Comment:
We can rename this to be without underscore since it's public `updateSpeedWithNewEpoch`. We can also change it to `external`

`function updateSpeedWithNewEpoch(CToken cToken) external {`